### ISSUE TITLE
Add project links

### ISSUE THREAD

**Issue by owiecc** [mapswipe/mapswipe#599]
Add a way of sharing a permalink to projects and a way to open these links in the app. This should help with project visibility.

**Comment by laurentS** [comment]
Thanks for suggesting this @owiecc. I'm not sure I fully understand though. Would you want a link to a live project so that tapping it bypasses the menu and gets you straight to the project page?

**Comment by owiecc** [comment]
Yes, something like that. Ultimately I would love to have a share button, that would allow me to share a specific task on Twitter so that I can ask people to help out.

**Comment by laurentS** [comment]
Related #39

**Comment by danbjoseph** [comment]
having links that also add some metadata to those contributions would be interesting. kind of like an affiliate link.  if you had an event you could send everyone a short link to use to open the app and it would also then let you see how much people contributed in the app sessions associated with opening the app through that link? is that even possible from a technical perspective?

**Comment by laurentS** [comment]
Relates to #617
